app/api/v2/handlers/operation_api.py

```diff
@@ -66,13 +66,26 @@ async def update_operation(self, request: web.Request):
         operation = await self.update_object(request)
         return web.json_response(operation.display)
 
-    @aiohttp_apispec.docs(tags=['operations'])
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Delete Operation',
+                          description='Delete an operation with confirmation from user. Returns with the OperationSchema format.')
```

Comment:
Returns with the OperationSchema format < likely redundant with line 72, also for "Delete an operation with confirmation from user" I think that's for the UI this shouldn't need a confirmation to delete, correct?